src/Microsoft.VisualStudio.Editors/PropPages/BuildPropPage.resx

```diff
@@ -1431,9 +1431,9 @@
   <data name="&gt;&gt;chkDefineDebug.ZOrder" xml:space="preserve">
     <value>23</value>
   </data>
-  <metadata name="$this.Localizable" type="System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
```

Comment:
This is moving from old style of representing this property, to the new one. It was automatically done by the designer.

Reply:
Did we recently update the designer to do this?

---
In reply to: [481792589](https://github.com/dotnet/project-system/pull/6517#discussion_r481792589) [](ancestors = 481792589)

Reply:
No this was either the transition from 1.0 -> 2.0 or from 3.5 -> 4.0. We just haven't opened the file in VS since then.